src/vma/sock/sockinfo_tcp.cpp

```diff
@@ -4409,12 +4409,7 @@ void sockinfo_tcp::tcp_tx_pbuf_free(void* p_conn, struct pbuf *p_buff)
 		mem_buf_desc_t * p_desc = (mem_buf_desc_t *)p_buff;
 
 		//potential race, ref is protected here by tcp lock, and in ring by ring_tx lock
-		if (likely(p_desc->lwip_pbuf_get_ref_count()))
```

Comment:
we should not remove the error message. but i agree that we can optimize this section